docs/content/operations/druid-console.md

```diff
@@ -86,5 +86,8 @@ The data servers tab lets you see the current status of the historical nodes and
 
 The SQL view lets you issue direct DruidSQL queries and display the results as a simple table. Note that despite the name this view also allows you to enter native Druid queries in Hjson format.
 
-![sql](./img/09-sql.png)
+![sql](./img/09-sql-1.png)
 
+The SQL view allows you to issue Druid's native query which is JSON over HTTP. To send Druid native query, you must start your query with { and in the format of JSON.
```

Comment:
nit:
>... issue Druid's native query which is JSON over HTTP. To send Druid native ...

This might read a bit smoother as "... issue queries in Druid's native query format, which is JSON over HTTP. To send a Druid native ..."